src/main/app/jurisdiction/jurisdictionRemovalHelper.ts

```diff
@@ -0,0 +1,28 @@
+import { stepsWithContentApplicant1 } from '../../steps';
+import { getAllPossibleAnswersForPath } from '../case/answers/possibleAnswers';
+import { Case } from '../case/case';
+
+export const getJurisdictionFieldsAsNull = (
```

Comment:
@lara-mpt should we call this setJurisdictionFieldsAsNull now?

Reply:
yeah that's better